docs/application/dotnet/guides/user-interface/nui/linear-layout.md

```diff
@@ -4,7 +4,9 @@
 
 ![LinearLayout](./media/linearTotal.png)
 
-`LinearAlignment` defines from where you can start positioning children. It can be used when children do not use all the space of a parent.
+`HorizontalAlignment` defines from where you can start positioning children from begin to end. It can be used when children do not use all the horizontal space of a parent.
```

Comment:
```suggestion
`HorizontalAlignment` defines where you can start positioning children from beginning to end. It can be used when children do not use all the horizontal space of a parent.
```